<!DOCTYPE html>

<html>

<head>
	<title>Capsid Constructors</title>

	<!-- IE Edge Meta Tag -->
	<meta http-equiv="X-UA-Compatible" content="IE=edge">

	<!-- Viewport -->
	<meta name="viewport" content="width=device-width, initial-scale=1">

	<!-- Bootstrap 4 Minified CSS -->
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">

	<!-- Custom CSS -->
	<link rel="stylesheet" href="/css/styles.css">

	<!-- Page Icon -->
	<link rel="icon" type="image/png" sizes="32x32" href="/favicon.png">

</head>

<body>

	<!-- Navigation bar placeholder-->
	<div id="nav-placeholder"></div>

	<!--Container to wrap page content-->
	<div class="container">

		<div class="row">
			<div class="col-xl-2 sidenavtracks">
				<div class="sidenav">
					<ul>
						<li><a href="#cloning" class="scroll">Cloning</a>
							<ul>
								<li><a href="#introduction" class="scroll">Introduction</a></li>

							</ul>
						</li>

					</ul>
				</div>
			</div>

			<div class="col-xl-10">

				<div id="cloning">
					<h1>Cloning</h1>
				</div>

				<img class="img-fluid" src="/images/cloning-1.png" />

				<div id="introduction">
					<h2>INTRODUCTION</h2>
				</div>

				<p>The process of turning protein design into functional units for use in analytical experiments requires that designed constructs, in the form the DNA sequences, are firstly cloned into vectors, and then expressed using <em>E. coli</em>. This section will describe the processes used to produce plasmids containing our mutated protein sequences. As outlined in our <u>Protein Deisgn</u> section (link to protein design), we have designed a number of HIV-1 CA protein variants. These mutations enable:</p>
				<ul>
					<li>Formation of disulfide bond cross-links between proteins which stabilise hexameric and pentameric formations,<sup>1</sup></li>
					<li>Inhibition of protein dimerization,<sup>2</sup></li>
					<li>Accessibility of residues for fluorophore conjugation, and </li>
					<li>Conjugation sites for reversible and irreversible chemistry.<sup>3,4</sup> </li>
				</ul>
				<p>We were kindly provided a number of base constructs by Derrick Lau, upon which we added mutations in order to achieve the desired protein capabilities. The following proteins were provided within pET11a plasmids, inserts of which were found between BamHI and Ndel restriction sites: </p>
				<ul>
					<li>Wild type HIV capsid protein (HIV-1 CA wild type)</li>
					<li>Disulphide bond stabilised hexamer (HIV-1 CA A14C/E45C)</li>
					<li>Disulphide bond stabilised hexamer with C-terminal dimer disruption (HIV-1 CA A14C/E45C/W184A/M185A)</li>
					<li>Disulphide bond stabilised pentamer (HIV-1 CA N21C/A22C)</li>
					<li>Disulphide bond stabilised pentamer with C-terminal dimer disruption (HIV-1 CA N21C/A22C/W184A/M185A)</li>
					<li>Accessible cysteine on the n-terminus flexible loop (HIV-1 CA K158C)</li>
				</ul>
				<p>Mutations were added to these constructs by three cloning methods, overviews of which have been summarised in Figure 1:</p>
				<ol>
					<li>Restriction-ligation cloning, for addition of a 6-Histadine tag to the C-terminal of the proteins</li>
					<li>Site directed mutagenesis, for single nucleotide polymorphisms, and</li>
					<li>Gibson assembly, for the generation of large mutations.</li>
				</ol>

				<img class="img-fluid" src="/images/cloning-2.png" />

				<p class="caption">Figure 1: overview of cloning methods used to achieve HIV-1 CA mutant proteins</p>

				<h2>Aims</h2>

				<p>To successfully clone mutated HIV-1 CA protein sequences into E. coli, using three cloning methods:</p>
				<ol>
					<li>Restriction-ligation cloning</li>
					<li>Site directed mutagenesis</li>
					<li>Gibson Assembly</li>
				</ol>
				<h2>Method 1: Restriction-ligation cloning</h2>

				<p>In order to add a polyhistidine tag to the C-terminus of each of our base constructs. The 6-Histidine tag added would enable the protein to for reversible bonds with our DNA templates through <u>Nickel-NTA conjugation chemistry</u> (link to conjugation chem section), for use in our kinetic experiments.<sup>3</sup> Our protocol for restriction-ligation cloning was adapted from Addgene.<sup>5</sup> We designed reverse primers with this encoded mutation, DNA sequences of which can be found <u>here</u> (link to supplementary materials). We then used these primers in a PCR amplification of our base constructs, and in doing so our mutations were added to the C-terminal ends of the proteins. </p>

				<img class="img-fluid" src="/images/cloning-3.png" />

				<p class="caption">Figure 2: Diagram illustrating the process of sequence insertion through PCR amplification</p>

				<p>These PCR amplified inserts, which now contained additional C-terminal base pairs, were visualised using gel electrophoresis, to check they were the correct size, the yields, and the purity. As we can see in Figure 3, our constructs ran at the expected height on the gel, so we proceeded to insert the amplified genes into our plasmid vector.</p>

				<div class="row tablegel">
					<img class="img-fluid gel" src="/images/cloning-3.png" />
					<table class="gel">
						<tbody>
							<tr>
								<td>
									<p>Lane</p>
								</td>
								<td>
									<p>Contents</p>
								</td>
								<td>
									<p>+/- result</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>1</p>
								</td>
								<td>
									<p>100bp DNA ladder</p>
								</td>
								<td>
								</td>
							</tr>
							<tr>
								<td>
									<p>2</p>
								</td>
								<td>
									<p>Wild-type + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>3</p>
								</td>
								<td>
									<p>K158C + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>4</p>
								</td>
								<td>
									<p>Pentamer + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>5</p>
								</td>
								<td>
									<p>Pentamer with dimer disruption + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>6</p>
								</td>
								<td>
									<p>Hexamer with dimer disruption + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
						</tbody>
					</table>
				</div>
			</div>


		</div>

		<!--Page footer-->
		<footer class="footer">
			<div class="container">
				<p>&copy; <strong>UNSW BIOMOD 2017 - Capsid Constructors.</strong></p>
			</div>
		</footer>

		<!-- jQuery -->
		<script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>

		<!-- Tether -->
		<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>

		<!-- Bootstrap Minified JavaScript -->
		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>

		<!-- Custom JavaScript, including navbar load -->
		<script type="text/javascript" src="/js/scripts.js"></script>

</body>



</html>
