<!DOCTYPE html>

<html>

<head>
	<title>Capsid Constructors</title>

	<!-- IE Edge Meta Tag -->
	<meta http-equiv="X-UA-Compatible" content="IE=edge">

	<!-- Viewport -->
	<meta name="viewport" content="width=device-width, initial-scale=1">

	<!-- Bootstrap 4 Minified CSS -->
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">

	<!-- Custom CSS -->
	<link rel="stylesheet" href="/css/styles.css">

	<!-- Page Icon -->
	<link rel="icon" type="image/png" sizes="32x32" href="/favicon.png">

</head>

<body>

	<!-- Navigation bar placeholder-->
	<div id="nav-placeholder"></div>

	<!--Container to wrap page content-->
	<div class="container">

		<div class="row">
			<div class="col-xl-2 sidenavtracks">
				<div class="sidenav">
					<ul>
						<li><a href="#cloning" class="scroll">Cloning</a>
							<ul>
								<li><a href="#introduction" class="scroll">Introduction</a></li>
								<li><a href="#aims" class="scroll">Aims</a></li>
								<li><a href="#methodone" class="scroll">Method 1</a></li>
								<li><a href="#methodtwo" class="scroll">Method 2</a></li>
								<li><a href="#methodthree" class="scroll">Method 3</a></li>
								<li><a href="#amplification" class="scroll">Amplification &amp; Sequencing</a></li>
								<li><a href="#results" class="scroll">Results</a></li>
								<li><a href="#discussion" class="scroll">Discussion</a></li>
								<li><a href="#references" class="scroll">References</a></li>
							</ul>
							<li><a href="/lab-book/protein-engineering/protein-design.html">Prev: Protein Design</a></li>
							<li><a href="/lab-book/protein-engineering/protein-production.html">Next: Protein Production</a></li>
						</li>

					</ul>
				</div>
			</div>

			<div class="col-xl-10">

				<div id="cloning">
					<h1>Cloning</h1>
				</div>

				<img class="img-fluid" src="/images/cloning-1.png" />

				<div id="introduction">
					<h2>INTRODUCTION</h2>
				</div>

				<p>The process of turning protein design into functional units for use in analytical experiments requires that designed constructs, in the form the DNA sequences, are firstly cloned into vectors, and then expressed using <em>E. coli</em>. This section will describe the processes used to produce plasmids containing our mutated protein sequences. As outlined in our <u>Protein Deisgn</u> section (link to protein design), we have designed a number of HIV-1 CA protein variants. These mutations enable:</p>
				<ul>
					<li>Formation of disulfide bond cross-links between proteins which stabilise hexameric and pentameric formations,<sup>1</sup></li>
					<li>Inhibition of protein dimerization,<sup>2</sup></li>
					<li>Accessibility of residues for fluorophore conjugation, and </li>
					<li>Conjugation sites for reversible and irreversible chemistry.<sup>3,4</sup> </li>
				</ul>
				<p>We were kindly provided a number of base constructs by Derrick Lau, upon which we added mutations in order to achieve the desired protein capabilities. The following proteins were provided within pET11a plasmids, inserts of which were found between BamHI and Ndel restriction sites: </p>
				<ul>
					<li>Wild type HIV capsid protein (HIV-1 CA wild type)</li>
					<li>Disulphide bond stabilised hexamer (HIV-1 CA A14C/E45C)</li>
					<li>Disulphide bond stabilised hexamer with C-terminal dimer disruption (HIV-1 CA A14C/E45C/W184A/M185A)</li>
					<li>Disulphide bond stabilised pentamer (HIV-1 CA N21C/A22C)</li>
					<li>Disulphide bond stabilised pentamer with C-terminal dimer disruption (HIV-1 CA N21C/A22C/W184A/M185A)</li>
					<li>Accessible cysteine on the n-terminus flexible loop (HIV-1 CA K158C)</li>
				</ul>
				<p>Mutations were added to these constructs by three cloning methods, overviews of which have been summarised in Figure 1:</p>
				<ol>
					<li>Restriction-ligation cloning, for addition of a 6-Histadine tag to the C-terminal of the proteins</li>
					<li>Site directed mutagenesis, for single nucleotide polymorphisms, and</li>
					<li>Gibson assembly, for the generation of large mutations.</li>
				</ol>

				<img class="img-fluid" src="/images/cloning-2.png" />

				<p class="caption">Figure 1: overview of cloning methods used to achieve HIV-1 CA mutant proteins</p>

				<div id="aims">
					<h2>Aims</h2>
				</div>

				<p>To successfully clone mutated HIV-1 CA protein sequences into E. coli, using three cloning methods:</p>
				<ol>
					<li>Restriction-ligation cloning</li>
					<li>Site directed mutagenesis</li>
					<li>Gibson Assembly</li>
				</ol>

				<div id="methodone">
					<h2>Method 1: Restriction-ligation cloning</h2>
				</div>

				<p>In order to add a polyhistidine tag to the C-terminus of each of our base constructs. The 6-Histidine tag added would enable the protein to for reversible bonds with our DNA templates through <u>Nickel-NTA conjugation chemistry</u> (link to conjugation chem section), for use in our kinetic experiments.<sup>3</sup> Our protocol for restriction-ligation cloning was adapted from Addgene.<sup>5</sup> We designed reverse primers with this encoded mutation, DNA sequences of which can be found <u>here</u> (link to supplementary materials). We then used these primers in a PCR amplification of our base constructs, and in doing so our mutations were added to the C-terminal ends of the proteins. </p>

				<img class="img-fluid" src="/images/cloning-3.png" />

				<p class="caption">Figure 2: Diagram illustrating the process of sequence insertion through PCR amplification</p>

				<p>These PCR amplified inserts, which now contained additional C-terminal base pairs, were visualised using gel electrophoresis, to check they were the correct size, the yields, and the purity. As we can see in Figure 3, our constructs ran at the expected height on the gel, so we proceeded to insert the amplified genes into our plasmid vector.</p>

				<div class="row tablegel">
					<img class="img-fluid gel" src="/images/cloning-4.png" />
					<table class="gel">
						<tbody>
							<tr>
								<td>
									<p>Lane</p>
								</td>
								<td>
									<p>Contents</p>
								</td>
								<td>
									<p>+/- result</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>1</p>
								</td>
								<td>
									<p>100bp DNA ladder</p>
								</td>
								<td>
								</td>
							</tr>
							<tr>
								<td>
									<p>2</p>
								</td>
								<td>
									<p>Wild-type + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>3</p>
								</td>
								<td>
									<p>K158C + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>4</p>
								</td>
								<td>
									<p>Pentamer + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>5</p>
								</td>
								<td>
									<p>Pentamer with dimer disruption + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>6</p>
								</td>
								<td>
									<p>Hexamer with dimer disruption + 6Histadine tag</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
						</tbody>
					</table>
				</div>

				<p class="caption">Figure 3: 1% agarose gel verifying successful amplification of HIV-1 CA proteins with C-terminal 6-Histidin additions</p>

				<p>The next step for creating functional plasmids to be used in protein expression was insertion of our new, mutated gene into our vector, pET11a (Figure 4).<sup>6</sup> This plasmid confers an ampicillin resistance gene (see the green arrow from base pairs 4609 to 5469, &lsquo;AmpR&rsquo;), which enables us to select for bacteria which have taken up this target plasmid during plasmid amplification and sequence verification. Our target cloning sites are located at restriction enzyme sites BamHI (319) and NdeI (359). Expression is under the control of the Lac operon (purple arrow), inducible by IPTG</p>

				<img class="img-fluid" src="/images/cloning-5.png" />

				<p class="caption">Figure 4: Plasmid Map of pET11a vector, sequence obtained from Addgene<sup>1</sup>, image generated using snapgene.</p>

				<p>We restriction digested both the gene inserts and vectors plasmids with BamHI and Ndel. This resulted in the creation of complementary &lsquo;sticky ends&rsquo;, which allow for the insert to be slotted into the plasmid. Enzymes were removed after the reaction using PCR clean up (Promega kit<sup>7</sup>). The inserts were then ligated into the plasmids, ready for amplification and subsequent sequence analysis. </p>

				<div id="methodtwo">
					<h2>Method 2: Site directed mutagenesis</h2>
				</div>
				<p>In order to make single nucleotide changes within our base constructs, we attempted site directed mutagenesis. Our protocol was adapted from methods proposed by Aligent Genomics.<sup>8</sup> We designed primers with additional nucleotides between complementary DNA to our target gene. Sequences for these primers can be found <u>here</u> (link to supplementary materials). PCR amplification of the entire plasmid was then performed using these primers, which results in the insertion of the new nucleotides into the gene of interest. </p>

				<img class="img-fluid" src="/images/cloning-6.png" />

				<p class="caption">Figure 5: diagram illustrating the process of site directed mutagenesis during PCR amplification of whole plasmid. </p>
				<p>The PCR reaction mixture was treated with DpnI restriction enzyme, which would break down plasmids which do not contain any mutation. These plasmids are identified as they are bacterially produced, causing methylation. Enzymes were removed after the reaction using PCR clean up kit.</p>
				<p>Insertion of our desired mutations into the plasmid were confirmed using gel electrophoresis (Figure 6). We performed a PCR optimisation, to ascertain the best conditions for performing site directed mutagenesis. Annealing temperatures for primer bindings were tested at 47<sup>o</sup>C, 52<sup>o</sup>C, 57<sup> o</sup>C, and 62<sup> o</sup>C. Additions of Magnesium, DMSO, and different polymerase enzymes were also screened. The brightest bands correlate to the highest yields of amplified plasmid. Figure 6 shows that our mutation was successfully introduced into the plasmid under all conditions, with the optimal condition across constructs a 57<sup>o</sup>C anneal using Taq polymerase enzyme. </p>

				<img class="img-fluid" src="/images/cloning-7.png" />

				<p class="caption">Figure 6: 1% agarose gel confirming addition of Spy Tag peptide sequence to the C-terminus of HIV-1 CA base plasmid constructs, using site directed mutagenesis. </p>

				<div id="methodthree">
					<h2>Method 3: Gibson assembly</h2>
				</div>
				<p>In order to introduce larger mutations to our plasmid constructs, such as Spy Catcher proteins, we used Gibson Assembly. Addition of a Spy Catcher protein at the C-terminal of our HIV-1 CA proteins would allow for irreversible Spy Catcher/Spy Tag conjugation to our DNA scaffolds.<sup>4</sup> Large pieces of DNA sequences, gBlocks, were designed to encode our desired mutations and synthesised by IDT. These were inserted into a pET C-terminal TEV His6 cloning vector with BioBrick polycistronic restriction sites (9bc), a gift from Scott Gradia (Addgene plasmid #48285).<sup>9</sup> This plasmid, as visualised in Figure 7, was chosen as it contains a gene for amplicilin resistance, labelled green &lsquo;AmpR&rsquo; on the plasmid map below, to allow for screening of successful transforms during plasmid amplification. The cloning site is found at restriction enzyme site HpaI (4135). The Lac operon found on a plasmid within our expression strain of <em>E. coli</em> is able to be induced by IPTG. This operon controls the expression of a T7 polymerase gene, which consequently binds to the T7 promoter site on pET9bc (4011), inducing expression of the insert at HpaI cloning site.</p>

				<img class="img-fluid" src="/images/cloning-8.png" />
				<p class="caption">Figure 7: Plasmid map of pET9bc, sequence obtained from Addgene,<sup>8</sup> image generated used Snapgene.</p>
				<p>Assembly of our recombinant plasmid was performed using the Gibson method,<sup>10</sup> as outlined in Figure 8. Our plasmid was linearised with Hpa1 restriction enzyme, then combined with our gBlocks and Gibson Assembly Master Mix. This was incubated for 30 minutes at 50<sup>o</sup>C. Generated plasmids were directly used for transformations into <em>E. coli</em>. </p>

				<img class="img-fluid" src="/images/cloning-9.png" />

				<p class="caption">Figure 8: Diagram illustrating the process of Gibson assembly sequence insertion into pET9bc plasmid<br> </p>
				<div id="amplification">
					<h2>Plasmid amplification and sequence verification</h2>
				</div>
				<p>After producing our recombinant plasmids, the next step was to transform these into our competent cells for plasmid amplification. We heat shock transformed our plasmids into a strain of <em>E. coli</em> optimised for cloning, DH5-alpha. This strain was chosen because it undergoes large-scale plasmid production, which we can then harvest for sequencing. After heat shock, our <em>E. coli</em> were plated onto LB agar containing ampicillin antibiotic. </p>
				<p>Due to the antibiotic resistance gene within our plasmid, only bacteria which had been successfully transformed should have grown on our plates. Colonies were selected from the plates, and a colony PCR performed to confirm successful transformation of our recombinant plasmids. Those colonies with the correct plasmids were grown up, and then a miniprep performed to extract and purify the plasmids. As a final confirmation of successful plasmid production with the correct mutation, sanger sequencing was performed by the Ramaciotti Centre for Genomics to verify gene sequences. An overview of this process is demonstrated in Figure 9.</p>
				<img class="img-fluid" src="/images/cloning-10.png" />
				<p class="caption">Figure 9: Overview of transformation and colony PCR to verify insertion, followed by plasmid harvesting and sequence verification to generate sequence level confirmation of mutation to our target gene. </p>
				<div id="results">
					<h2>RESULTS</h2>
				</div>
				<h3>Restriction-Ligation cloning</h3>
				<p>We have successfully introduced 6-Histadine tags to the C-terminus all of our base constructs via restriction-ligation cloning. This has enabled us to produce HIV-1 CA proteins which can be conjugated to our DNA origami using <u>Nickle/NTA chemistry</u> (link to conj. chem). The presence of our target inserts was confirmed by performing colony PCR (Figures 10, 11 and 12), to amplify the gene of interest from whole colonies. This produced an amplified band of DNA, with a size corresponding to our target gene, verified by comparison to the ladder. This was repeated on several colonies per construct. </p>

				<img class="img-fluid gel" src="/images/cloning-11.png" />

				<table class="center gel">
					<tbody>
						<tr>
							<td>
								<p>Lane</p>
							</td>
							<td>
								<p>Contents</p>
							</td>
							<td>
								<p>+/- result</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>1</p>
							</td>
							<td>
								<p>100bp DNA ladder</p>
							</td>
							<td>

							</td>
						</tr>
						<tr>
							<td>
								<p>2</p>
							</td>
							<td rowspan="5">
								<p>Wild-type HIV-1 CA protein with C-terminal 6-Histadine tag</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>3</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>4</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>5</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>6</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>7</p>
							</td>
							<td rowspan="5">
								<p>HIV-1 CA protein with available Cysteine on flexible loop and C-terminal 6-Histadine tag</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>8</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>9</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>10</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>11</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>12</p>
							</td>
							<td rowspan="5">
								<p>Cross-linked pentameric HIV-1 CA protein with C-terminal 6-Histadine tag</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>13</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>14</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>15</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>16</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>18</p>
							</td>
							<td>
								<p>Uncut pET11a plasmid</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
					</tbody>
				</table>

				<p class="caption">Figure 10: 1% agarose gel of Colony PCR on C-terminal 6-Histidine mutants.</p>

				<img class="img-fluid gel" src="/images/cloning-12.png" />

				<table class="center gel">
					<tbody>
						<tr>
							<td>
								<p>Lane</p>
							</td>
							<td>
								<p>Contents</p>
							</td>
							<td>
								<p>+/- result</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>1</p>
							</td>
							<td>
								<p>100bp DNA ladder</p>
							</td>
							<td>

							</td>
						</tr>
						<tr>
							<td>
								<p>2</p>
							</td>
							<td rowspan="5">
								<p>Cross-linked pentameric HIV-1 CA protein with C-terminal 6-Histidine tag and dimer distruption</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>3</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>4</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>5</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>6</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>7</p>
							</td>
							<td rowspan="5">
								<p>Cross-linked hexameric HIV-1 CA protein with C-terminal 6-Histidine tag</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>8</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>9</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>10</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>11</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>12</p>
							</td>
							<td rowspan="5">
								<p>Cross-linked hexameric HIV-1 CA protein with C-terminal 6-Histidine tag and dimer distruption</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>13</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>14</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>15</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>16</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
					</tbody>
				</table>

				<p class="caption">Figure 11: 1% agarose gel of Colony PCR on C-terminal 6-Histidine mutants.</p>

				<div class="tablegel">
					<img class="img-fluid gel" src="/images/cloning-13.png" />
					<table class="gel">
						<tbody>
							<tr>
								<td>
									<p>Lane</p>
								</td>
								<td>
									<p>Contents</p>
								</td>
								<td>
									<p>+/- result</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>1</p>
								</td>
								<td>
									<p>100bp DNA ladder</p>
								</td>
								<td>

								</td>
							</tr>
							<tr>
								<td>
									<p>2</p>
								</td>
								<td rowspan="5">
									<p>Hexameric HIV-1 CA protein with C-terminal 6-Histidine tag,</p>
									<p>Colonies A-E </p>
									<p>(re-digested)</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>3</p>
								</td>
								<td>
									<p>-</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>4</p>
								</td>
								<td>
									<p>-</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>5</p>
								</td>
								<td>
									<p>-</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>6</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
						</tbody>
					</table>
				</div>

				<p class="caption">Figure 12:1% agarose gel of Colony PCR on Hexameric HIV-1 CA protein constructs with C-terminal 6-Histidine.</p>

				<h3>Site Directed Mutagenesis</h3>

				<p>Although we were able to produce correctly sized constructs during PCR amplification, we were unable to successfully transform these vectors into our expression strain, as shown in our unsuccessful colony PCRs below (Figure 13). We believe that our issues with this cloning technique may have been due to issues in either digestion or transformation of the plasmids. For more information on this process please refer to our <a href="#">supplementary materials</a>. In the interest of time, we decided to focus our efforts towards Gibson Assembly as an alternate method of introducing our mutations.</p>

				<img class="img-fluid gel" src="/images/cloning-14.png" />

				<table class="center gel">
					<tbody>
						<tr>
							<td>
								<p>Lane</p>
							</td>
							<td>
								<p>Contents</p>
							</td>
							<td>
								<p>+/- result</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>1</p>
							</td>
							<td>
								<p>100bp DNA ladder</p>
							</td>
							<td>

							</td>
						</tr>
						<tr>
							<td>
								<p>2</p>
							</td>
							<td rowspan="5">
								<p>Hexameric HIV-1 CA protein with C-terminal Spy Tag peptide,</p>
								<p>Colonies A-E</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>3</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>4</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>5</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>6</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>7</p>
							</td>
							<td rowspan="5">
								<p>Hexameric HIV-1 CA protein with dimer disruption and C-terminal Spy Tag peptide,</p>
								<p>Colonies A-E</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>8</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>9</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>10</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>15</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>16</p>
							</td>
							<td rowspan="5">
								<p>Pentameric HIV-1 CA protein with C-terminal Spy Tag peptide,</p>
								<p>Colonies A-E</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>17</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>18</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>19</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>20</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
					</tbody>
				</table>

				<p class="caption">Figure 13: 1% agarose gel of Colony PCR on C-terminal Spy Tag mutants.</p>

				<h3>Gibson Assembly</h3>

				<p>We were also successful in creating a number of important recombinant plasmid using the Gibson Assembly method. Due to the time restraints of the competition, we decided to focus on a select number of mutated proteins which would be most useful for our analysis experiments. See below our colony PCR gels confirming insertion of our gBlocks into our plasmids (Figures 14, 15, and 16). </p>

				<img class="img-fluid gel" src="/images/cloning-15.png" />
				
				<table class=" center gel">
						<tbody>
							<tr>
								<td>
									<p>Lane</p>
								</td>
								<td>
									<p>Contents</p>
								</td>
								<td>
									<p>+/- result</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>1</p>
								</td>
								<td>
									<p>1kb DNA ladder</p>
								</td>
								<td>

								</td>
							</tr>
							<tr>
								<td>
									<p>2</p>
								</td>
								<td rowspan="8">
									<p>Wild type protein with dimer disruption, C-terminal Spy Catcher, and 6-Histadine tag under TEV restriction site:</p>
									<p>Colonies A &ndash; H </p>
								</td>
								<td>
									<p>-</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>3</p>
								</td>
								<td>
									<p>-</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>4</p>
								</td>
								<td>
									<p>+ </p>
								</td>
							</tr>
							<tr>
								<td>
									<p>5</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>6</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>7</p>
								</td>
								<td>
									<p>+</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>8</p>
								</td>
								<td>
									<p>-</p>
								</td>
							</tr>
							<tr>
								<td>
									<p>9</p>
								</td>
								<td>
									<p>-</p>
								</td>
							</tr>
						</tbody>
					</table>
				
				<p class="caption">Figure 14: 1% agarose gel of Colony PCR on Gibson assembly of Wild-Type HIV-1 CA with dimer disruption and C-terminal Spy Catcher protein.</p>

				<img class="img-fluid gel" src="/images/cloning-16.png" />

				<table class="center gel">
					<tbody>
						<tr>
							<td>
								<p>Lane</p>
							</td>
							<td>
								<p>Contents</p>
							</td>
							<td>
								<p>+/- result</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>1</p>
							</td>
							<td>
								<p>100bp DNA ladder</p>
							</td>
							<td>

							</td>
						</tr>
						<tr>
							<td>
								<p>2</p>
							</td>
							<td rowspan="8">
								<p>Wild type protein with dimer disruption, C-terminal 6-Histidine tag under TEV restriction site:</p>
								<p>Colonies A &ndash; H </p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>3</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>4</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>5</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>6</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>7</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>8</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>9</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>10</p>
							</td>
							<td rowspan="7">
								<p>HIV-1 CA protein with accessible cysteine on flexible loop, dimer distruption, and C-terminal 6-Histidine tag under TEV restriction site:</p>
								<p>Colonies A &ndash; H</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>11</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>12</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>13</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>14</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>15</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>16</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
					</tbody>
				</table>

				<p class="caption">Figure 15: 1% agarose gel of Colony PCR on Gibson Assemblies of Wild type protein with dimer disruption and C-terminal 6-Histidine tag, and HIV-1 CA protein with accessible cysteine on flexible loop, dimer distruption and C-terminal 6-Histidine tag.</p>

				
				<img class="img-fluid gel" src="/images/cloning-17.png" />
				
				<table class="center gel">
					<tbody>
						<tr>
							<td>
								<p>Lane</p>
							</td>
							<td>
								<p>Contents</p>
							</td>
							<td>
								<p>+/- result</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>1</p>
							</td>
							<td>
								<p>1kb DNA ladder</p>
							</td>
							<td>

							</td>
						</tr>
						<tr>
							<td>
								<p>3</p>
							</td>
							<td rowspan="8">
								<p>HIV-1 CA protein with accessible cysteine on flexible loop, dimer distruption, C-terminal Spy Catcher protein, and 6-Histidine tag under TEV restriction site</p>
							</td>
							<td>
								<p>+</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>4</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>5</p>
							</td>
							<td>
								<p>- </p>
							</td>
						</tr>
						<tr>
							<td>
								<p>6</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>7</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>8</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>9</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
						<tr>
							<td>
								<p>10</p>
							</td>
							<td>
								<p>-</p>
							</td>
						</tr>
					</tbody>
</table>
					
				<p class="caption">Figure 16: 1% agarose gel of Colony PCR on Gibson Assembly of HIV-1 CA protein with accessible cysteine on flexible loop, dimer distruption, C-terminal Spy Catcher protein, and 6-Histidine tag</p>

				<div id="discussion">
					<h2>Discussion</h2>
				</div>

				<p>Of the 36 possible plasmids containing mutated HIV-1 CA protein sequences we could potentially have generated, 16 were produced and positive sequencing results confirmed. Ideally, all constructs would have been made to allow for a number of different aspects of capsid assembly to be tested, but due to the time restraints of the competition this was no feasible. We were, however, successful in producing the most important constructs for preliminary kinetic and structural analysis. Below is a list of all our successful plasmid constructs, with their associated potential experimental uses. </p>

				<h3>Base Constructs</h3>



				<h3>C-Terminal mutations</h3>
				<ul>
					<li>HIV-1 CA Wild-type protein with C-terminal 6-Histadine tag</li>
					<li>HIV-1 CA protein with K158C substitution and C-terminal 6-Histadine tag</li>
					<li>HIV-1 CA cross-linked hexameric protein with C-terminal 6-Histadine tag</li>
					<li>HIV-1 CA HIV-1 CA cross-linked hexameric protein with dimer disruption and C-terminal 6-Histadine tag</li>
					<li>HIV-1 CA cross-linked pentameric protein with C-terminal 6-Histadine tag</li>
					<li>HIV-1 CA HIV-1 CA cross-linked pentameric protein with dimer disruption and C-terminal 6-Histadine tag</li>
				</ul>
				<p>Addition of a 6-Histidine tag will allow for each protein to be reversibly conjugated to our DNA scaffolds.</p>

				<h3>Gibson Assemblies</h3>
				<ul>
					<li>Wild type HIV capsid protein (HIV-1 CA wild type) - Positive control for analysis experiments</li>
					<li>Disulphide bond stabilised hexamer (HIV-1 CA A14C/E45C) - Used for testing lattice assembly and analysing hexamer-hexamer interaction</li>
					<li>Disulphide bond stabilised hexamer with C-terminal dimer disruption (HIV-1 CA A14C/E45C/W184A/M185A) - Used for testing hexameric affinity to DNA</li>
					<li>Disulphide bond stabilised pentamer (HIV-1 CA N21C/A22C) - Used For testing lattice assembly and analysing pentamer-hexamer interaction</li>
					<li>Disulphide bond stabilised pentamer with C-terminal dimer disruption (HIV-1 CA N21C/A22C/W184A/M185A) - Used for testing pentameric affinity to DNA</li>
					<li>Accessible cysteine on the n-terminus flexible loop (HIV-1 CA K158C) - Accessible cysteine for conjugation to fluorophores, for use in single molecule imaging experiments</li>
				</ul>

				<p>With our desired mutations now successfully cloned into plasmids, our constructs were ready for expression. <a href="/lab-book/protein-engineering/protein-production.html">Click here</a> to read about how we expressed our proteins! (link to protein production page)</p>

				<div id="references">
					<h2>References</h2>
				</div>

				<ol>
					<li>Pornillos, O., Ganser-Pornillos, B., Kelly, B., Hua, Y., Whitby, F., Stout, C., Sundquist, W., Hill, C. and Yeager, M. (2009). X-Ray Structures of the Hexameric Building Block of the HIV Capsid. <em>Cell</em>, 137(7), pp.1282-1292.</li>
					<li>Jacques, D., McEwan, W., Hilditch, L., Price, A., Towers, G. and James, L. (2016). HIV-1 uses dynamic capsid pores to import nucleotides and fuel encapsidated DNA synthesis. <em>Nature</em>, 536(7616), pp.349-353.</li>
					<li>Hochuli, E., Bannwarth, W., D&ouml;beli, H., Gentz, R. and St&uuml;ber, D. (1988). Genetic Approach to Facilitate Purification of Recombinant Proteins with a Novel Metal Chelate Adsorbent. <em>Nature Biotechnology</em>, 6(11), pp.1321-1325.</li>
					<li>Li, L., Fierer, J., Rapoport, T. and Howarth, M. (2014). Structural Analysis and Optimization of the Covalent Association between SpyCatcher and a Peptide Tag. <em>J Mol Biol</em>, 426(2), pp.309-317.</li>
					<li>Addgene, Plasmid Cloning by PCR, <a href="https://www.addgene.org/protocols/pcr-cloning/">https://www.addgene.org/protocols/pcr-cloning/</a></li>
					<li>Addgene, Vector Database, Plasmid: pET11a, <a href="https://www.addgene.org/vector-database/2625/">https://www.addgene.org/vector-database/2625/</a></li>
					<li>Promega, Wizard SV Gel and PCR Clean-Up system, <a href="https://www.promega.com.au/resources/protocols/technical-bulletins/101/wizard-sv-gel-and-pcr-cleanup-system-protocol/">https://www.promega.com.au/resources/protocols/technical-bulletins/101/wizard-sv-gel-and-pcr-cleanup-system-protocol/</a></li>
					<li>Aligent Genomics, QuikChange II Site-Directed Muagenesis Kit, <a href="https://www.agilent.com/cs/library/usermanuals/public/200523.pdf">https://www.agilent.com/cs/library/usermanuals/public/200523.pdf</a></li>
					<li>Addgene, pET C-terminal TEV His6 cloning vector with BioBrick polycistronic restriction sites (9bc), <a href="https://www.addgene.org/48285/">https://www.addgene.org/48285/</a></li>
					<li>Gibson DG, Young L, Chuang RY, Venter JC, Hutchison CA 3rd, Smith HO (2009). Enzymatic assembly of DNA molecules up to several hundred kilobases.<em>Nature Methods</em>,<strong>6</strong>(5), pp. 343&ndash;345.</li>
				</ol>

			</div>

		</div>

		<!--Page footer-->
		<footer class="footer">
			<div class="container">
				<p>&copy; <strong>UNSW BIOMOD 2017 - Capsid Constructors.</strong></p>
			</div>
		</footer>

		<!-- jQuery -->
		<script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>

		<!-- Tether -->
		<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>

		<!-- Bootstrap Minified JavaScript -->
		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>

		<!-- Custom JavaScript, including navbar load -->
		<script type="text/javascript" src="/js/scripts.js"></script>

</body>



</html>
